eox_tenant/signals.py

```diff
@@ -45,6 +45,12 @@ def _update_settings(domain):
 
     if not config.get("EDNX_USE_SIGNAL"):
         LOG.info("Site %s, does not use eox_tenant signals", domain)
+        # Some django apps need to be reinitialized
```

Comment:
We could have this block and the next block that does the same refactored into a function.

However EDNX_TENANT_INSTALLED_APPS will never exist for sites that do not have EDNX_USE_SIGNAL unless we set that in the settings.common module.
I would also suggest that if a site does not have ednx_use_signal we should ignore the repopulate apps procedure. We should not expect a site to work properly in any case if it does not use signals

Reply:
I added this block of code in this part due to a failing test (test__repopulate_apps_called)

 I am going to fix that!